components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -161,6 +161,13 @@ void MAMMicrophysics::set_grids(
   // Downwelling solar flux at the surface [w/m2]
   add_field<Required>("SW_flux_dn", scalar3d_int, W / m2, grid_name);
 
+  // Diagnostic fluxes          
+  constexpr int gas_pcnst = mam4::gas_chemistry::gas_pcnst;
+  const FieldLayout vector2d_nmodes =
+      grid_->get_2d_vector_layout(nmodes, "mam4::gas_chemistry::gas_pcnst");
+  add_field<Computed>("dqdt_so4_aqueous_chemistry", vector2d_nmodes, kg/kg/s,  grid_name);
+  add_field<Computed>("dqdt_h2so4_uptake", vector2d_nmodes, kg/kg/s,  grid_name);
```

Comment:
We need to change the units here to `kg/m2/s` for both of these diagnostics.

Reply:
Done.